src/main/java/de/sakpaas/backend/service/OccupancyService.java

```diff
@@ -71,8 +76,14 @@ public static Double calculateAverage(List<Occupancy> occupancies, ZonedDateTime
    * @param x the x value
    * @return the y value
    */
-  public static double bellCurve(double x) {
-    return FACTOR_B * Math.exp(-Math.pow(-x - 15, 2) / FACTOR_A);
+  @VisibleForTesting
+  double calculateFactor(double x) {
```

Comment:
Ich weiß ich bin heute kleinlich, aber vlt in calculateWeightFactor umbennen oder so? ^^

Reply:
Dann eher `calculateAccumulationFactor`